modular_zubbers/code/modules/synths/code/synth_prefab.dm

```diff
@@ -0,0 +1,17 @@
+/mob/living/carbon/human/species/synth
+	race = /datum/species/synthetic
+
+// Used for printing dead synth bodies
+/mob/living/carbon/human/species/synth/empty
+
+/mob/living/carbon/human/species/synth/empty/Initialize(mapload)
+	. = ..()
+	var/old_death_sound
+	death_sound = null // We don't need them screaming
+	death()
+
+	for(var/obj/item/organ/internal/brain/B in organs)
```

Comment:
should do var = locate (type) in organs instead

Reply:
wilco